infrastructure/src/main/java/org/corfudb/infrastructure/ManagementServer.java

```diff
@@ -544,12 +544,7 @@ public void shutdown() {
             corfuRuntime.shutdown();
         }
 
-        try {
-            failureDetectorService.awaitTermination(serverContext.SHUTDOWN_TIMER.getSeconds(),
-                    TimeUnit.SECONDS);
-        } catch (InterruptedException ie) {
-            log.debug("failureDetectorService awaitTermination interrupted : {}", ie);
-        }
+        failureDetectorService.shutdownNow();
```

Comment:
We are already performing a `failureDetectorService.shutdownNow()` in the beginning of this method.